app/scripts/classes/DepthyViewer.js

```diff
@@ -477,7 +477,6 @@ Copyright (c) 2014 Rafał Lindemann. http://panrafal.github.com/depthy
         if (depth.useAlpha) {
           // move inverted alpha to rgb, set alpha to 1
           depthTextureSprite.filters.push(invertedAlphaToRGBFilter);
-          depthTextureSprite.filters = depthTextureSprite.filters;
```

Comment:
Yep, seems to work just fine. :)

On Tue, Nov 24, 2015 at 3:53 AM, Rafał Lindemann <notifications@github.com>
wrote:

> In app/scripts/classes/DepthyViewer.js
> <https://github.com/panrafal/depthy/pull/31#discussion_r45725779>:
>
> > @@ -477,7 +477,6 @@ Copyright (c) 2014 Rafał Lindemann. http://panrafal.github.com/depthy
> >          if (depth.useAlpha) {
> >            // move inverted alpha to rgb, set alpha to 1
> >            depthTextureSprite.filters.push(invertedAlphaToRGBFilter);
> > -          depthTextureSprite.filters = depthTextureSprite.filters;
>
> AFAIR it was set on purpose, are you sure it works correctly now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/panrafal/depthy/pull/31/files#r45725779>.
>



-- 
Mikhail Yurasov